api/util/importer.js

```diff
@@ -4,23 +4,37 @@ var locations = rfr('lambda/locations');
 var resources = rfr('lambda/resources');
 var bookings = rfr('lambda/bookings');
 var cognito = rfr('util/cognito');
-var logger = rfr('util/logger');
+
+
+var SampleGroups = [
+  {
+    name: 'adminGroup',
+    description: 'Cognito user group for administrators',
+    precedence: 0,
+    iam: 'something'
+  },
+  {
+    name: 'clientGroup',
+    description: 'Cognito user group for spacefinder users',
+    precedence: 0,
```

Comment:
Shouldn't precedence for the client group be different than the admin group? I would think admin group should have a lower precedence value than the client group, since admins are also users or likely would be listed as such in many apps?